src/Appwrite/Utopia/Response/Model/HealthTime.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class HealthTime extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('remote', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'Current unix timestamp on trustful remote server.',
+                'default' => 0,
+                'example' => 1639490751,
+            ])
+            ->addRule('local', [
```

Comment:
```suggestion
            ->addRule('localTime', [
```

Reply:
Another alternative can be `sourceTime` and `destinationTime`

Reply:
remote and local makes sense to me. Remote is external trusted source, local is mine server